core/site.go

```diff
@@ -114,6 +114,9 @@ type Site struct {
 	batteryMode              api.BatteryMode // Battery mode (runtime only, not persisted)
 	batteryModeExternal      api.BatteryMode // Battery mode (external, runtime only, not persisted)
 	batteryModeExternalTimer time.Time       // Battery mode timer for external control
+
+	smartFeedinDisableLimit  *float64 // Feed-in limit
```

Comment:
Do we really need a price value here or isn't the desired behavior always to avoid export if feedin price drops to negative? So could this just be a bool to enable/disable the feature?

Reply:
https://github.com/evcc-io/evcc/issues/21747#issuecomment-2974171790

Reply:
Lets make this (and the naming) dependent on https://github.com/evcc-io/evcc/issues/21747#issuecomment-2974181278

Reply:
Based on the few comments so far I'd suggest we start with a simple on/off implementation to disable feedin when feedin rates get negative. Margins and fees can already be adjusted in tariff configuration.

This would make the implementation (specially ui) a lot simpler.

Reply:
Already is disable-only

Reply:
I mean the price threshold. It currently accepts a float:
https://github.com/evcc-io/evcc/pull/21839/files#diff-9dc8ed5b73297cf3c3d1d32cb36520c8f982a0194fad1e7b3fc521c5799e2023R160

Reply:
I‘d leave it to the user, especially as it comes with the disadvantage of breaking pv mode. You may not do this for just 1ct?

Reply:
Lets start with limited frontend then?